Closure_Front_End/src/huji-import/CandidateCourses.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+
+<h1 class="title">קורסים לייבוא</h1>
+<div class="notification is-danger" v-if="courses.length === 0">
+    מתוך
+    <strong>{{totalCourseCount}}</strong>
+    קורסים שנקלטו מאתר האוניברסיטה, לא אותרו קורסים אשר נתמכים על ידי האתר.
+</div>
+<div class="box" v-if="courses.length > 0">
+<div class="notification is-success">
+  אותרו
+  <strong>{{courses.length}}</strong>
+  קורסים שניתן לייבא
+</div>
+<div class="notification is-warning" v-if="ambiguousCourses.length > 0">
+  עבור
+  <strong>{{ambiguousCourses.length}}</strong>
+  מהקורסים, יש לבחור ידנית את הסמסטר
+</div>
+  <table class="table">
+    <thead>
+      <tr>
+        <th>מספר קורס</th>
+        <th>שם קורס</th>
+        <th>נקודות זכות</th>
+        <th>סמסטר</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr v-for="course in courses" :key="course.course_id" 
+          :class="{ 'has-background-warning-light': course.ambiguous }">
+        <td>{{ course.course_id }}</td>
+        <td>{{ course.name }}</td>
+        <td>{{ course.points }}</td>
+        <td>
+            <div class="control">
+            <label class="radio">
+                <input type="radio" id="one" value="FIRST" v-model="course.semester">
+                א'
+            </label>
+            <label class="radio">
+                <input type="radio" id="two" value="SECOND" v-model="course.semester">
+                ב'
+            </label>
+            </div>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+
+    <div class="block">
+    <div class="notification is-danger is-light">
+      <strong>אזהרה:</strong>
+      <br/>
+      עם ייבוא הקורסים, כל הקורסים הקודמים ששמורים בדפדפן יימחקו
```

Comment:
Isn't there a way to add the new courses without overriding? This could be a bit frustrating

Reply:
Duplicates are only counted once, so I think allowing them won't do much harm (given students may retake courses. Showing duplicates isn't a huge advantage, but I think it's simpler to just let them be, wdyt?)

An "override?" checkbox sounds like a good solution.

Reply:
I added a choice between merging and overriding(with a warning message in case we're overriding)